src/components/ButtonComponent/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {ReactComponent as ButtonPlus} from '../../assets/images/buttonplus.svg'
```

Comment:
Imports need to be terminated
```suggestion
import {ReactComponent as ButtonPlus} from '../../assets/images/buttonplus.svg';
```